src/modules/imageresizer/tests/Properties/SettingsTests.cs

```diff
@@ -186,5 +186,93 @@ public void IDataErrorInfo_Item_returns_empty_when_not_JpegQualityLevel()
 
             Assert.Empty(result);
         }
-    }
+
+        [Fact]
+        public void Reload_createsFile_when_FileNotFound()
+        {
+            var settings = new Settings();
+            Settings.SettingsPath = ".\\test_settings.json";
+            if (System.IO.File.Exists(Settings.SettingsPath))
+            {
+                System.IO.File.Delete(Settings.SettingsPath);
+            }
+
+            Assert.False(System.IO.File.Exists(Settings.SettingsPath));
+            settings.Reload();
+            Assert.True(System.IO.File.Exists(Settings.SettingsPath));
+            if (System.IO.File.Exists(Settings.SettingsPath))
+            {
```

Comment:
Moved the initial changing settings.json path section to the SettingsTests constructor and moved the If File Exists Delete logic to the Dispose. I didnt add the If File Exists Delete logic in the constructor since it should be deleted at the end with the dispose method. The constructor and Dispose in the class are called per test in xunit but the constructor and Dispose called in the base class fixture (AppFixture in this case) are called only once, so any one-time setup and one-time dispose can be done there.